ngraph/test/frontend/paddlepaddle/requirements_dev.txt

```diff
@@ -1,5 +1,6 @@
 # PaddlePaddle - generate test models
 paddlepaddle==2.1.0
+paddledet>=2.1.0
```

Comment:
Why do we need this additional dependency?

Reply:
Yes, we do. both paddlepaddle and paddledet are asked just to generate test models. paddle nms api is in paddledet.

Reply:
@ilyachur do you have more concern about this requirement? I  am not sure who should I notify to avoid CI and e2e validation fail because of missing paddledetection package.